internal/pkg/cli/job_deploy.go

```diff
@@ -113,11 +114,7 @@ func (o *deployJobOpts) Ask() error {
 	if err := o.askEnvName(); err != nil {
 		return err
 	}
-	tag, err := askImageTag(o.imageTag, o.prompt, o.cmd)
-	if err != nil {
-		return err
-	}
-	o.imageTag = tag
+	o.imageTag = imageTagFromGit(o.cmd, o.imageTag)
```

Comment:
That makes sense! I've done this, I'll send a follow-up PR to move `git.go` inside the `exec` pkg. So that we have one package for running any commands, and increase our unit test coverage by moving it outside of the `cli` pkg.